native/src/components/PoiDetails.tsx

```diff
@@ -75,11 +76,12 @@ const PoiDetails: React.FC<PoiDetailsProps> = ({
   const { location, address, postcode, town } = poi.location
   const { distance, title } = feature.properties
 
-  // TODO refactor to use a get coordinates method IGAPP-806
   const onNavigate = () => {
-    if (location && poi.featureLocation?.geometry.coordinates) {
-      const navigationUrl = getNavigationDeepLinks(location, poi.featureLocation.geometry.coordinates)
-      Linking.openURL(navigationUrl).catch(() => showSnackbar(t('navigationApplication')))
+    if (location) {
```

Comment:
Remove this check

Reply:
If I the check  `if (location)` I get this error: Argument of type 'string | null | undefined' is not assignable to parameter of type 'string'.

Reply:
yes, but you don't need to pass that, see comment above. you already pass the full model, no need to pass one property again

Reply:
oh you're right